matchbox_demo/Cargo.toml

```diff
@@ -24,17 +24,16 @@ console_error_panic_hook = "0.1"
 console_log = "0.2"
 serde_qs = "0.10"
 wasm-bindgen = "0.2"
-ggrs = { version = "0.9", features = ["wasm-bindgen"] }
 
 [dependencies]
 matchbox_socket = { path = "../matchbox_socket", features = ["ggrs-socket"] }
-bevy = { version = "0.8", default-features = false }
-ggrs = "0.9"
-bevy_ggrs = "0.10"
+bevy = { version = "0.9", default-features = false }
+ggrs = { version = "0.9.3", features = ["wasm-bindgen"]  }
```

Comment:
Is there a reason to add the `wasm-bindgen` feature to non-wasm tagets?

Reply:
nope, this was just me not reading properly